functions-exercises.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by daniel.potts on 20/08/15.
+ */
+/*
+ * Created by daniel.potts on 20/08/15.
+ */
+//Fuel calculator: for a given MPG and distance, calculate the number of gallons of fuel needed for a trip.
+var gallons  = function (MPG, distance){
+    var gallonsrequired = (distance / MPG);
+    console.log(gallonsrequired +" Gallons required");};
+gallons(68,5);
+//Write a love calculator:
+var length = function (name1, name2)
+{
+    var nameLength1 = name1.length,
+        nameLength2 =name2.length;
+    var result = ((nameLength1 +nameLength2) - (Math.random()* 6));
+    result *=42;
+    if (result >100) {
+        result = 100;
+        console.log ("Your match is " + result + "%");}
+    else {
+        console.log("Your match is " + Math.round(result) + "%");}};
```

Comment:
Again, function should return a number, it is not part of its responsibility to output.